hassio/src/components/hassio-card-content.ts

```diff
@@ -43,15 +43,15 @@ class HassioCardContent extends LitElement {
       ${this.iconImage
         ? html`
             <div class="icon_image ${this.iconClass}">
-              <img src="${this.iconImage}" title="${this.iconTitle}" />
+              <img src="${this.iconImage}" title="${this.iconTitle == null ? "" : this.iconTitle}" />
```

Comment:
Use the `ifDefined` directive <https://lit-html.polymer-project.org/guide/template-reference#ifdefined>
```suggestion
              <img src="${this.iconImage}" title="${ifDefined(this.iconTitle)}" />
```

Reply:
Btw, `null` and `undefined` are 2 different things, your check would not have fixed this as the value is `undefined` and not `null`.

`${this.iconTitle || ""}` would have fixed it because `null` and `undefined` are both falsy.

Reply:
`this.iconTitle || ""` clobbers all falsy values, e.g. `0`. Maybe no one is using any falsy values anywhere -- I was unable to do a check for that. Doing a non-strict comparison (`==`) to `null` WILL match both `null` and `undefined`.

Reply:
Oh right, missed the non strict check, we used to not allow those...